runtime/src/server/middleware/get_page_handler.ts

```diff
@@ -351,7 +351,7 @@ export function get_page_handler(
 				.replace('%sapper.html%', () => html)
 				.replace('%sapper.head%', () => head)
 				.replace('%sapper.styles%', () => styles)
-				.replace('%sapper.cspnonce%', () => nonce_value);
+				.replace(/%sapper.cspnonce%/g, nonce_value);
```

Comment:
```suggestion
				.replace(/%sapper\.cspnonce%/g, () => nonce_value);
```

The `.` should be escaped, and the `() =>` should be preserved as it's there to avoid `.replace`'s special behavior with `$1`, `$2`, etc.